JsonSchema/ReadOnlyKeyword.cs

```diff
@@ -44,7 +44,7 @@ internal class ReadOnlyKeywordJsonConverter : JsonConverter<ReadOnlyKeyword>
 	{
 		public override ReadOnlyKeyword Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
 		{
-			if (reader.TokenType != JsonTokenType.True || reader.TokenType != JsonTokenType.False)
+			if (reader.TokenType != JsonTokenType.True && reader.TokenType != JsonTokenType.False)
```

Comment:
I'd probably invert the condition altogether, would be a bit easier to read IMO.